code/modules/antagonists/changeling/powers/chameleon_skin.dm

```diff
@@ -1,7 +1,7 @@
 /datum/action/changeling/chameleon_skin
 	name = "Chameleon Skin"
 	desc = "Our skin pigmentation rapidly changes to suit our current environment. Costs 25 chemicals."
-	helptext = "Allows us to become invisible after a few seconds of standing still. Can be toggled on and off."
+	helptext = "Allows us to become invisible after a few seconds of standing still. Can be toggled on and off. Better than the mutation."
```

Comment:
This is weirdly worded imo.
Relies on the user having knowledge on something they might have never seen yet.